src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not an SCSS file. Please change `href="styles/index.scss"` to `href="styles/index.css"` to ensure the stylesheet is correctly linked.